docs/source/docker/getting_started/first_steps.rst

```diff
@@ -640,11 +640,10 @@ The output directory is structured in the following way:
 * data:
    The data directory contains everything to do with data. 
    
-    * If `enable_corruptness_check=True`, it will contain a "clean" version of the dataset. 
-    * If `remove_exact_duplicates=True`, it will contain a copy of the `embeddings.csv` 
-      where all duplicates are removed. Otherwise, it will simply store the 
-      embeddings computed by the model.
-    * If `selected_sequence_length > 1`, it will contain a `sequence_information.json`
+    * `embeddings.csv` contains the computed embeddings for all new input samples used in selection.
```

Comment:
> all new input samples

It also contains the datapool samples. Furthermore, it does not include the corrupt samples. Does it include the duplicate samples?

Reply:
corrected